### ISSUE TITLE
OperationalError: no such column: format

### ISSUE THREAD

**Issue by Julioevm** [ggozad/oterm#15]
Hi,

First, I wanna say that this project is very promising, good job!

Now the issue, I just installed on a Mac M1 through home-brew and when I try to run it I get the error: `OperationalError: no such column: format`

Here's the stack trace:
<details><summary>Stracktrace</summary>

```
╭───────────────────── Traceback (most recent call last) ──────────────────────╮
│ /opt/homebrew/Cellar/oterm/0.1.12/libexec/lib/python3.11/site-packages/oterm │
│ /app/oterm.py:108 in on_mount                                                │
│                                                                              │
│   105 │                                                                      │
│   106 │   async def on_mount(self) -> None:                                  │
│   107 │   │   self.store = await Store.create()                              │
│ ❱ 108 │   │   saved_chats = await self.store.get_chats()  # type: ignore     │
│   109 │   │   if not saved_chats:                                            │
│   110 │   │   │   self.action_new_chat()                                     │
│   111 │   │   else:                                                          │
│                                                                              │
│ ╭────────────────────── locals ───────────────────────╮                      │
│ │ self = OTerm(title='oTerm', classes={'-dark-mode'}) │                      │
│ ╰─────────────────────────────────────────────────────╯                      │
│                                                                              │
│ /opt/homebrew/Cellar/oterm/0.1.12/libexec/lib/python3.11/site-packages/oterm │
│ /store/store.py:159 in get_chats                                             │
│                                                                              │
│   156 │   │   tuple[int, str, str, list[int], str | None, str | None, Litera │
│   157 │   ]:                                                                 │
│   158 │   │   async with aiosqlite.connect(self.db_path) as connection:      │
│ ❱ 159 │   │   │   chats = await chat_queries.get_chats(connection)  # type:  │
│   160 │   │   │   chats = [                                                  │
│   161 │   │   │   │   (id, name, model, json.loads(context), template, syste │
│   162 │   │   │   │   for id, name, model, context, template, system, format │
│                                                                              │
│ ╭─────────────────────────── locals ───────────────────────────╮             │
│ │ connection = <Connection(Thread-5, started 6214348800)>      │             │
│ │       self = <oterm.store.store.Store object at 0x106b87e10> │             │
│ ╰──────────────────────────────────────────────────────────────╯             │
│                                                                              │
│ /opt/homebrew/Cellar/oterm/0.1.12/libexec/lib/python3.11/site-packages/aiosq │
│ l/queries.py:98 in afn                                                       │
│                                                                              │
│    95                                                                        │
│    96 def _make_async_fn(fn: QueryFn) -> QueryFn:                            │
│    97 │   async def afn(self: Queries, conn, *args, **kwargs):               │
│ ❱  98 │   │   return await fn(self, conn, *args, **kwargs)                   │
│    99 │                                                                      │
│   100 │   return _query_fn(afn, fn.__name__, fn.__doc__, fn.sql, fn.operatio │
│   101                                                                        │
│                                                                              │
│ ╭───────────────────────────────── locals ─────────────────────────────────╮ │
│ │   args = ()                                                              │ │
│ │   conn = <Connection(Thread-5, started 6214348800)>                      │ │
│ │     fn = <function _make_sync_fn.<locals>.fn at 0x106aa2c00>             │ │
│ │ kwargs = {}                                                              │ │
│ │   self = Queries(['delete_chat', 'delete_chat_cursor', 'get_chat',       │ │
│ │          'get_chat_cursor', 'get_chats', 'get_chats_cursor',             │ │
│ │          'get_messages', 'get_messages_cursor', 'rename_chat',           │ │
│ │          'rename_chat_cursor', 'save_chat', 'save_chat_cursor',          │ │
│ │          'save_context', 'save_context_cursor', 'save_message',          │ │
│ │          'save_message_cursor'])                                         │ │
│ ╰──────────────────────────────────────────────────────────────────────────╯ │
│                                                                              │
│ /opt/homebrew/Cellar/oterm/0.1.12/libexec/lib/python3.11/site-packages/aiosq │
│ l/adapters/aiosqlite.py:24 in select                                         │
│                                                                              │
│   21 │   │   return sql                                                      │
│   22 │                                                                       │
│   23 │   async def select(self, conn, _query_name, sql, parameters, record_c │
│ ❱ 24 │   │   async with conn.execute(sql, parameters) as cur:                │
│   25 │   │   │   results = await cur.fetchall()                              │
│   26 │   │   │   if record_class is not None:                                │
│   27 │   │   │   │   column_names = [c[0] for c in cur.description]          │
│                                                                              │
│ ╭───────────────────────────────── locals ─────────────────────────────────╮ │
│ │  _query_name = 'get_chats'                                               │ │
│ │         conn = <Connection(Thread-5, started 6214348800)>                │ │
│ │   parameters = ()                                                        │ │
│ │ record_class = None                                                      │ │
│ │         self = <aiosql.adapters.aiosqlite.AioSQLiteAdapter object at     │ │
│ │                0x1069d2cd0>                                              │ │
│ │          sql = 'SELECT id, name, model, context, template, system,       │ │
│ │                format FROM chat;'                                        │ │
│ ╰──────────────────────────────────────────────────────────────────────────╯ │
│                                                                              │
│ /opt/homebrew/Cellar/oterm/0.1.12/libexec/lib/python3.11/site-packages/aiosq │
│ lite/context.py:39 in __aenter__                                             │
│                                                                              │
│   36 │   │   return self._coro.__await__()                                   │
│   37 │                                                                       │
│   38 │   async def __aenter__(self) -> _T:                                   │
│ ❱ 39 │   │   self._obj = await self._coro                                    │
│   40 │   │   return self._obj                                                │
│   41 │                                                                       │
│   42 │   async def __aexit__(self, exc_type, exc, tb) -> None:               │
│                                                                              │
│ ╭──────────────────────── locals ─────────────────────────╮                  │
│ │ self = <aiosqlite.context.Result object at 0x106b00b40> │                  │
│ ╰─────────────────────────────────────────────────────────╯                  │
│                                                                              │
│ /opt/homebrew/Cellar/oterm/0.1.12/libexec/lib/python3.11/site-packages/aiosq │
│ lite/core.py:190 in execute                                                  │
│                                                                              │
│   187 │   │   """Helper to create a cursor and execute the given query."""   │
│   188 │   │   if parameters is None:                                         │
│   189 │   │   │   parameters = []                                            │
│ ❱ 190 │   │   cursor = await self._execute(self._conn.execute, sql, paramete │
│   191 │   │   return Cursor(self, cursor)                                    │
│   192 │                                                                      │
│   193 │   @contextmanager                                                    │
│                                                                              │
│ ╭───────────────────────────────── locals ─────────────────────────────────╮ │
│ │ parameters = ()                                                          │ │
│ │       self = <Connection(Thread-5, started 6214348800)>                  │ │
│ │        sql = 'SELECT id, name, model, context, template, system, format  │ │
│ │              FROM chat;'                                                 │ │
│ ╰──────────────────────────────────────────────────────────────────────────╯ │
│                                                                              │
│ /opt/homebrew/Cellar/oterm/0.1.12/libexec/lib/python3.11/site-packages/aiosq │
│ lite/core.py:133 in _execute                                                 │
│                                                                              │
│   130 │   │                                                                  │
│   131 │   │   self._tx.put_nowait((future, function))                        │
│   132 │   │                                                                  │
│ ❱ 133 │   │   return await future                                            │
│   134 │                                                                      │
│   135 │   async def _connect(self) -> "Connection":                          │
│   136 │   │   """Connect to the actual sqlite database."""                   │
│                                                                              │
│ ╭───────────────────────────────── locals ─────────────────────────────────╮ │
│ │     args = (                                                             │ │
│ │            │   'SELECT id, name, model, context, template, system,       │ │
│ │            format FROM chat;',                                           │ │
│ │            │   ()                                                        │ │
│ │            )                                                             │ │
│ │       fn = <built-in method execute of sqlite3.Connection object at      │ │
│ │            0x106b81e40>                                                  │ │
│ │ function = functools.partial(<built-in method execute of                 │ │
│ │            sqlite3.Connection object at 0x106b81e40>, 'SELECT id, name,  │ │
│ │            model, context, template, system, format FROM chat;', ())     │ │
│ │   future = <Future finished exception=OperationalError('no such column:  │ │
│ │            format')>                                                     │ │
│ │   kwargs = {}                                                            │ │
│ │     self = <Connection(Thread-5, started 6214348800)>                    │ │
│ ╰──────────────────────────────────────────────────────────────────────────╯ │
│                                                                              │
│ /opt/homebrew/Cellar/oterm/0.1.12/libexec/lib/python3.11/site-packages/aiosq │
│ lite/core.py:106 in run                                                      │
│                                                                              │
│   103 │   │   │   │   break                                                  │
│   104 │   │   │   try:                                                       │
│   105 │   │   │   │   LOG.debug("executing %s", function)                    │
│ ❱ 106 │   │   │   │   result = function()                                    │
│   107 │   │   │   │   LOG.debug("operation %s completed", function)          │
│   108 │   │   │   │                                                          │
│   109 │   │   │   │   def set_result(fut, result):                           │
│                                                                              │
│ ╭───────────────────────────────── locals ─────────────────────────────────╮ │
│ │      function = functools.partial(<built-in method close of              │ │
│ │                 sqlite3.Connection object at 0x106b81e40>)               │ │
│ │        future = <Future finished result=None>                            │ │
│ │        result = None                                                     │ │
│ │          self = <Connection(Thread-5, started 6214348800)>               │ │
│ │ set_exception = <function Connection.run.<locals>.set_exception at       │ │
│ │                 0x106b63100>                                             │ │
│ │    set_result = <function Connection.run.<locals>.set_result at          │ │
│ │                 0x106b63060>                                             │ │
│ ╰──────────────────────────────────────────────────────────────────────────╯ │
╰──────────────────────────────────────────────────────────────────────────────╯
OperationalError: no such column: format```

</details>

**Comment by ggozad** [comment]
Hey Julio, I thought I fixed that in 0.1.12 on Saturday. Could you verify you are running that version with `oterm --version`

**Comment by ggozad** [comment]
Could you please try again and let me know if things work?
